style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  margin-right: 500px;
+  margin-left: 500px;
+  margin-top: 0;
+  min-height: 50vh;
```

Comment:
- [ ] Kindly remove this code from your style sheet, it is causing your project to be unresponsive.
- [ ] Use the developer tools in your browser to convert your site to mobile view when developing. checkout this link for more information on how to convert your site into a mobile view[Link](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome)

Once you remove the CSS use the developer tools in your browser to follow the Figma design. Kindly style individual elements so you can see the changes as you code.

Reply:
wow, these correction has pretty much improved my code. 
Thank you so much @Munsa1

Reply:
Excellent, you just reminded me of this, thanks